src/common/snippets/docs/debug_capabilities/LIR_passes_serialization.md

```diff
@@ -0,0 +1,20 @@
+# Snippets LIR passes serialization
+
+LIR(Linear Intermediate Representation) is used as graph reprsentation in control flow pipeline, where dozens of passes are applied to LIR. This is to transfer the graph gradually to the stage that can generate kernel directly via expression instruction emission. When each pass is applied to LIR, there are some expected changes. Developers maybe want to check if the result is as expected. This capability is introduced to serialize LIRs before and after passes, then developer can check these LIR changes and status.
+
+To turn on snippets LIR passses serialization feature, the following environment variable should be used:
+```sh
+    OV_SNIPPETS_DUMP_LIR=<space_separated_options> binary ...
+```
+
+Examples:
+```sh
+    OV_SNIPPETS_DUMP_LIR="passes=ExtractLoopInvariants dir=path/dumpdir formats=all" binary ...
+    OV_SNIPPETS_DUMP_LIR="passes=all dir=path/dumpdir formats=control_flow" binary ...
+    OV_SNIPPETS_DUMP_LIR="passes=Insert formats=data_flow" binary ...
+```
+
+Option names are case insensitive, the following options are supported:
+ - `passes` : Dump LIR around the pass if the value is pass name or is included in pass name. Key word 'all' means to dump LIR around every pass. This option is a must have, should not be omitted.
```

Comment:
Can I dump lir after several passes? For example, does `OV_SNIPPETS_DUMP_LIR="passes=ExtractLoopInvariants,InsertLoops` work?
If it doesn't work, is it easy and clear how to implement it?
If it already work - it's really great! Could you please update the doc to highlight this moment?

Reply:
Now we support `OV_SNIPPETS_DUMP_LIR="passes=ExtractLoopInvariants,InsertLoops` and document is also updated!
Note that I introduce and use `MultipleStringPropertySetter`, which is immune for future pass add, delete and modify. alternatively using `BitsetFilterPropertySetter` need update this debug_cap code once there are new/delete/modify to existing pass.
Thanks Alexandra!

Reply:
It's really great! Thanks a lot, Chenhu!